### PR TITLE
Sync primitives — Part 1

### PR DESCRIPTION
**Pull request by radex**
Some basic preparation/clean up for sync primitives:

- move existing helpers around + add tests
- add basic sync helper (unused yet) + tests

### CODE REVIEW

**Comment by radex** [comment]
@kokusGr CR please :)

File: src/sync/helpers.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type { ColumnName } from '..'
+import type { RawRecord } from '../RawRecord'
+
+export function addToRawSet(rawSet: ?string, value: string): string {
+  const array = rawSet ? rawSet.split(',') : []
+  const set = new Set(array)
+  set.add(value)
+  return Array.from(set).join(',')
```
Comment (kokusGr) [inline src/sync/helpers.js]: Hmm this seems like a lot of work. Maybe we should think about other soultion. But I guess first we need to test it's performance ;) Anyway it's not the most important thing to do right now.

**Comment by kokusGr** [comment]
LGTM :) But there is a conflict

File: src/sync/helpers.js (hunk 1 above)
Comment (radex) [reply src/sync/helpers.js]: this code is unchanged, just moved to a different place ;) I'll be tweaking performance once the whole sync infra is done.

Comment (kokusGr) [reply src/sync/helpers.js]: Yes, i know just wanted to point that out ;)

**Comment by mobily** [comment]
thanks @radex! :)

**Pull request merged**